examples/windows/dll/BUILD

```diff
@@ -4,18 +4,69 @@ filegroup(
     visibility = ["//examples:__pkg__"],
 )
 
+cc_library(
+    name = "hello-library-header",
+    hdrs = ["hello-library.h"],
+)
+
 cc_binary(
     name = "hellolib.dll",
     srcs = [
         "hello-library.cpp",
     ],
+    deps = [
+        ":hello-library-header",
+    ],
+    # Define COMPILING_DLL to export symbols during compiling the DLL.
+    # See hello-library.h
+    copts = ["/DCOMPILING_DLL"],
     linkshared = 1,
 )
 
+
+# **Explicitly link to hellolib.dll**
+
+# Declare hellolib.dll as data dependency and load it explicitly in code.
 cc_binary(
-    name = "hello",
+    name = "hello-explicitly",
```

Comment:
I'm confused by this name; the word "explicitly" doesn't explain enough for me. Could you choose a different name, which is more concrete? How about e.g. "hello_load-dll-at-runtime" or something similar?

Reply:
Same here, addressed in the internal change.